fractal-first/backend/src/main/java/me/amasiero/osb/OsbApplication.java

```diff
@@ -0,0 +1,13 @@
+package me.amasiero.osb;
+
```

Comment:
Question: Should we change the package for YanchWare?

Reply:
Totally up to you. It is your work, we can keep it as is or move it into a yanchware package ;)

Reply:
If you don't see any problem! It's okay to  me, happy to collaborate some how :)